cpr/session.cpp

```diff
@@ -291,6 +293,25 @@ void Session::Impl::SetVerifySsl(const VerifySsl& verify) {
     }
 }
 
+void Session::Impl::SetProtocolVersion(const ProtocolVersion& protocolversion) {
```

Comment:
Prefer lowerCamelCase for all variable names, so `protocolVersion` is more consistent here